src/main/java/kr/allcll/backend/support/exception/AllcllErrorCode.java

```diff
@@ -17,7 +17,8 @@ public enum AllcllErrorCode {
     BASKET_NOT_FOUND("관심과목 정보가 존재하지 않습니다."),
 
     EXTERNAL_CONNECTION_TERMINATED("외부 서버와의 연결이 종료되었습니다."),
-    ;
+    EMITTER_NOT_FOUND("존재하지 않는 SSE 연결입니다"),
+    CONNECTION_DENIED("현재 SSE 연결을 할 수 없습니다.");
```

Comment:
```suggestion
    SSE_EMITTER_NOT_FOUND("존재하지 않는 SSE 연결입니다"),
    SSE_CONNECTION_DENIED("현재 SSE 연결을 할 수 없습니다."),
    ;
```

SSE 라는 prefix가 붙어있으면 에러 코드 간에 구분이 쉬울 것 같아요! 
그리고 enum 상수를 정의할 때에 각 줄은 `,`로 끝나고 `;`는 개행 아래에 두는 것이 좋을 것 같아요. 이러면 line change 수가 줄어들거든요!

Reply:
저도 line change 수 줄이는거 좋아하구 그래서 enum에 ;를 개행 아래에 두고 싶었는데요, 라인포맷팅하면 자꾸 위로 알아서 올라가요. 그래서 그냥 위에 두었어요...